server/routes/post.routes.ts

```diff
@@ -1,13 +1,10 @@
 import { Router } from "express";
-import { getPosts, getFavoritePosts, getIsFavorite, setFavorite } from "../controllers/post.controller";
-import { existsPost, existsUser } from '../middlewares/validate-existance';
-import validateJWT from "../middlewares/validate-jwt";
+import { getPosts, getFavoritePosts, getIsFavorite, getMyPosts } from "../controllers/post.controller";
 const router = Router();
 
-router.get('/', [validateJWT], getPosts);
-router.get('/user/:userId', [validateJWT], getFavoritePosts);
-router.get('/isFavorite/post/:postId/user/:userId', [existsPost, existsUser, validateJWT], getIsFavorite);
-
-router.put('/setFavorite/post/:postId/user/:userId', [existsPost, existsUser, validateJWT], setFavorite);
+router.get('/', getPosts);
```

Comment:
These routes should have the corresponding validators. (Middlewares)

Reply:
I will add it in the next pr